trustpoint/onboarding/api.py

```diff
@@ -364,6 +365,8 @@ def revoke(request: HttpRequest, device_id: int) -> tuple[int, dict] | HttpRespo
     if not device:
         return 404, {'error': 'Device not found.'}
 
-    if device.revoke_ldevid():
-        return 200, {'success': True}
+    if device.ldevid:
+        if device.revoke_ldevid(ReasonCode.CESSATION):
```

Comment:
Do we want to set the revocation reason in the request or is this fine?

Reply:
In the long term I think that makes sense, this is more of a placeholder. Since we will possibly move from Ninja DRF though, I don't know if it makes sense to add it now. Definitely adding a comment to keep it in mind though, the reason should not be hardcoded for the API.